docs/antrea-network-policy.md

```diff
@@ -545,6 +546,10 @@ with `namespaces` field.
 specifies both namespaceSelector and podSelector selects particular Pods within
 particular Namespaces.
 
+**nodeSelector**: This selects particular Nodes in cluster. The selected Node's
+IPs will set as "sources" if `nodeSelector` set in `ingress` section, or as
```

Comment:
will set as -> will be used as

set in -> is set in the

Reply:
Done.